docs/apb_cli.md

```diff
@@ -262,6 +262,24 @@ Create binding out of secret `foo` and add it to Deployment Config `bar`
 apb binding add foo bar
 ```
 
+Our example APBs create secrets that match the name of the APB pod. So if you want to bind Postgresql APB to Mediawiki, you would first provision Postgresql (`apb bundle provision postgresql-apb`) and Mediawiki (`apb bundle provision mediawiki-apb`). Once they are done, you should see a secret named `bundle-<hash>` if you do `oc get secret`. Then find the name of the DeploymentConfig you want to bind to (`oc get dc`).If the DeploymentConfig is `mediawiki-1234` a binding command may look like
```

Comment:
Once they are done, 
->
Once the APBs have finished provisioning

So if you want to bind Postgresql APB to Mediawiki
->
To bind Postgresql APB to Mediawiki

if you do
->
when you run

I might consider putting this paragraph into numbered list format instead of a bunch of "and then" type statements.